example/hello/odp_hello.c

```diff
@@ -24,29 +24,23 @@
 #include <odp_api.h>
```

Comment:
Includes related to Linux PID and CPU affinity can be now removed from above:

/* Linux CPU affinity */
#define _GNU_SOURCE
#include <sched.h>

/* Linux PID */
#include <sys/types.h>
#include <unistd.h>

Reply:
Ack